crates/katana/core/Cargo.toml

```diff
@@ -62,7 +61,6 @@ messaging = [
     "alloy-provider",
     "alloy-rpc-types",
     "alloy-signer",
-    "alloy-signer-wallet",
```

Comment:
@kariy can you confirm that this is not needed.

Reply:
Alloy crates are a bit modified in #1870, but may adjusted during a rebase.